src/javascript/app_2/Utils/Date/date-time.js

```diff
@@ -17,11 +17,9 @@ export const epochToMoment = epoch => moment.unix(epoch).utc();
 export const toMoment = value => {
     if (!value) return moment().utc(); // returns 'now' moment object
     if (value instanceof moment && value.isValid() && value.isUTC()) return value; // returns if already a moment object
-    const is_number      = typeof value === 'number';
-    // need to explicitly convert date string to a JS Date object then pass that into Moment
-    // to get rid of the warning: Deprecation warning: moment construction falls back to js Date
-    const formatted_date = moment(new Date(value)).format('YYYY-MM-DD');
-    return is_number ? epochToMoment(value) : moment.utc(formatted_date);
+    if (typeof value === 'number') return epochToMoment(value); // returns epochToMoment() if not a date
+
+    return moment.utc(value);
```

Comment:
@msamprz please make sure it doesn't show depreciation warning in the console. and also works across browsers (i think moment.js tend to behaves differently in different browsers)

Reply:
yep, for sure 👍